build.gradle

```diff
@@ -5,6 +5,7 @@ buildscript {
   dependencies {
     classpath 'com.android.tools.build:gradle:2.2.2'
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.13.0'
+    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
```

Comment:
Interesting, though it looks like (from the docs) you can only do this if you're using Jack & Jill. Is that false?  http://tools.android.com/tech-docs/jackandjill#TOC-Repackaging-support